Lib/sysconfig.py

```diff
@@ -465,10 +465,14 @@ def _get_sysconfigdata_name():
         f'_sysconfigdata_{sys.abiflags}_{sys.platform}_{multiarch}',
     )
 
+def _print_config_dict(d, stream):
+    print (" {", file=stream)
```

Comment:
```suggestion
    print ("{", file=stream)
```

One space is written already via `f.write('build_time_vars = ')`.